### DISCUSSION TITLE
guide/multiple-class-convert

### DISCUSSION THREAD

**Discussion by giscus** [linpeilie/mapstruct-plus#17] [Announcements]
# guide/multiple-class-convert

MapStructPlus 一个类与多个类之间转换 multiple class convert

https://mapstruct.plus/guide/multiple-class-convert.html

**Comment by Ruan-WenBo** [comment]
API是什么的？

**Reply by Ruan-WenBo** [reply]
一对多是要调用多次convert方法吗
